.travis.yml

```diff
@@ -1,70 +1,57 @@
 # Travis can building for Linux and macOS
-matrix:
+language: python
+addons:
+  apt:
+    update: true
+    packages: libxkbcommon-x11-0  # Linux package needed for Qt to work
+  homebrew:
+    update: true
+    packages: freetype
+
+services: xvfb
+python:
+  - 3.5
+  - 3.6
+  - 3.7
+jobs:
+  allow_failures:
+    - python: 3.8
+    - os: osx
   include:
-    - os: linux
-      dist: xenial 
-      sudo: required
-      language: python
-      python: 3.5
-    - os: linux
-      dist: xenial 
-      sudo: required
-      language: python
-      python: 3.6
-    - os: linux
-      dist: xenial
-      sudo: required
-      language: python
-      python: 3.7
-    # To maximise compatibility pick earliest image, OS X 10.12
+    - python: 3.8
+      before_install: pip install pygame==2.0.0.dev6
     - os: osx
-      osx_image: xcode8.3
-      sudo: required
+      osx_image: xcode11.2
       language: generic
-      python: 3.6
+      python: 3.7
+      before_install: ln -s /usr/local/bin/pip3 /usr/local/bin/pip
 
-before_install:
+install:
```

Comment:
I think the original intent here was to do all "info commands" in the `before_install`, and only the actually installation in the `install` entries.
Is there is a reason not to use `before_install`?
I don't mind either way, just wondering if there is something I wasn't aware of.

Reply:
When different OSes need different install steps, I tend to use __before_install__ for OS-dependent steps and __install__ for steps that are the same on all OSes.

Reply:
Oh, that's a nice convention! Sounds good.